manifests/plugin.pp

```diff
@@ -77,7 +77,7 @@
     $proxy_port  = undef,
 ) {
 
-  include elasticsearch
+  require elasticsearch
```

Comment:
This seems to have caused a dependency cycle.

```
 Error: Could not apply complete catalog: Found 1 dependency cycle:
  (Augeas[defaults_es-01] => Elasticsearch::Service::Init[es-01] => Elasticsearch::Service[es-01] => Elasticsearch::Instance[es-01] => Class[Elasticsearch] => Elasticsearch::Plugin[mobz/elasticsearch-head/1.0.0] => Elasticsearch_plugin[mobz/elasticsearch-head/1.0.0] => Elasticsearch::Plugin[mobz/elasticsearch-head/1.0.0] => Class[Elasticsearch])
```

Don't think we need the require for accessing the variable ( plugindir ) since it always worked with the restart_on_change one.

Also in https://github.com/elastic/puppet-elasticsearch/blob/master/manifests/init.pp#L411 there is already an explicit dependency for it.